common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleServiceMetrics.java

```diff
@@ -153,24 +153,6 @@ private static ShuffleServiceMetricsInfo getShuffleServiceMetricsInfoForGenericV
       valueName + " value of " + baseName);
   }
 
-  private static class ShuffleServiceMetricsInfo implements MetricsInfo {
```

Comment:
`static class` is equivalent to `record`? Specifically, `static` part.

Reply:
Yes, Modifier 'static' is redundant for inner records